src/components/App/App.js

```diff
@@ -1,33 +1,56 @@
 import React, { Component } from "react";
 import Header from "../Header/Header.js";
 import Movies from "../Movies/Movies.js";
-import { movieData, selectedMovieDetails } from "../../testData.js";
 import MovieDetails from "../MovieDetails/MovieDetails.js";
 
 class App extends Component {
   constructor() {
     super();
 
     this.state = {
-      movies: movieData.movies,
-      selectedMovie: '',
+      movies: [],
+      selectedMovie: "",
+      error: ""
     };
   }
 
-  showMovieDetails = () => {
-    this.setState({ selectedMovie: selectedMovieDetails.movie });
+  componentDidMount() {
+    fetch("http://rancid-tomatillos.herokuapp.com/api/v2/movies")
```

Comment:
I'm sure you already have done this, but if not - it's good practice to move fetch calls into their own file :D